rabbitmq/default.toml

```diff
@@ -17,10 +17,11 @@ default_user="guest"
 default_pass="guest"
 
 [rabbitmq.management]
-
 # Whether to enable the rabbitmq management plugin.
 enabled=false
 
 [erlang]
+cookie=false
```

Comment:
Why did this change from a string to a boolean?

Reply:
> The erlang cookie default is now false. Unless we're clustering, we don't need to render a cookie, and we shouldn't have a preconfigured cookie.

If we use a string, it will set the cookie to that string no matter whether we're clustering or not. We've made it a best practice across other projects (like community chef-cookbooks) to not use pre-configured secrets.